### ISSUE TITLE
Forgeessentials Wont work

### ISSUE THREAD

**Issue by jamiejp** [ForgeEssentials/ForgeEssentials#2096]
I downloaded this version forgeessentials-1.8-1.8.3.54 in ma server file mods folder, but when checked it, it dont work, no commands working, (MC Version is 1.8.9) Pweese help. Also im new to this mod.

**Comment by spacebuilder2020** [comment]
As it says on the jenkens page that build does not support 1.8.9.

There is no 1.8.9 build released yet.

**Comment by jamiejp** [comment]
o but why some1 said on curse forge thats it works with 1.8.9

**Comment by DomAmato** [comment]
This release on my branch works with 1.8.9 but I have no intention of maintaining everything or feature requests as that is the purpose of the main project here. The forl is for my own purposes so there is no guarantee that a feature you use will be 100% functional.

https://github.com/CityOfLearning/ForgeEssentials/releases/tag/dyn.v1.8.9

**Comment by luacs1998** [comment]
@DomAmato https://github.com/ForgeEssentials/ForgeEssentials/tree/189/develop you can build this

**Comment by DomAmato** [comment]
No but I looked at the Travis build and you are having the same problem I did. The new mixin stuff breaks the annotation processing needed for the playerlogger for some reason. If you revert it back to the old files it will compile (the mixins from the 1.8 branch on here.) I was going to try and figure it out but never had the time and the old mixins still worked.

Dominic Amato
Human Computer Interaction Masters Candidate
Research Assistant: Digital Youth Network (http://digitalyouthnetwork.org)
Lab Manager: Technology for Social Good (http://tsg.cdm.depaul.edu)
Lab Associate: Physical Computing Lab (http://goo.gl/dguGzt)
DePaul University

-----Original Message-----
